telegram/ext/dispatcher.py

```diff
@@ -469,7 +478,8 @@ def dispatch_error(self, update, error):
 
         Args:
             update (:obj:`str` | :class:`telegram.Update` | None): The update that caused the error
-            error (:class:`telegram.TelegramError`): The Telegram error that was raised.
+            error (:class:`telegram.TelegramError` | :Type:`Exception`): The (Telegram) error
+            that was raised.
```

Comment:
just :obj:`Exception` will do. No need for both

Reply:
good point